src/node_builtins.cc

```diff
@@ -139,6 +139,9 @@ BuiltinLoader::BuiltinCategories BuiltinLoader::GetBuiltinCategories() const {
         "sqlite",  // Experimental.
         "sys",     // Deprecated.
         "wasi",    // Experimental.
+#if !HAVE_SQLITE
```

Comment:
Silly question: why is this a negation?

Reply:
Interesting. This file follows this same pattern. I'm curious 🤔

Reply:
This is the list of internal modules which are not requireable. If we have SQLite available it is requireable and should not be in the list.

Reply:
Oh, thank you so much!